spec/factories.rb

```diff
@@ -54,4 +54,12 @@
       data_url "I'm a number!"
     end
   end
+
```

Comment:
i think it would be good to have a factory for comment - would like to figure out how to properly make that.